hasher-matcher-actioner/hmalib/lambdas/api/actions_api.py

```diff
@@ -0,0 +1,115 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+import bottle
+import typing as t
+from dataclasses import dataclass, asdict
+from .middleware import jsoninator, JSONifiable, DictParseable
+from hmalib.common.config import HMAConfig, update_config
+from hmalib.common import config as hmaconfig
+from hmalib.common.actioner_models import ActionPerformer
+
+
+@dataclass
+class Action(JSONifiable, DictParseable):
+    name: str
+    config_subtype: str
+    url: str
+    headers: str
```

Comment:
Can we make these `parameters` a dict so that they can have different fields for differnt action performer subtypes?